### PR TITLE
Added a setting to change the line number color

### PR DESCRIPTION
**Pull request by Paulb23**
Added a setting to change the line number color.

### CODE REVIEW

**Comment by bojidar-bg** [comment]
@Paulb23 ~ um... colors again?

**Comment by Paulb23** [comment]
@bojidar-bg Can't go wrong with more colors :P don't worry i'm almost finished with colorising the text editor... for the moment.

File: tools/editor/plugins/script_editor_plugin.cpp [hunk 1]
```diff
@@ -288,6 +288,7 @@ void ScriptTextEditor::_load_theme_settings() {
 
 	get_text_edit()->set_custom_bg_color(EDITOR_DEF("text_editor/background_color",Color(0,0,0,0)));
 	get_text_edit()->add_color_override("font_color",EDITOR_DEF("text_editor/text_color",Color(0,0,0)));
+	get_text_edit()->add_color_override("line_number_color",EDITOR_DEF("text_editor/line_number_color",Color(0,0,0)));
```
Comment (akien-mga) [inline tools/editor/plugins/script_editor_plugin.cpp]: Which setting takes the priority by default? ``Color::html(66aaaaaa)`` or ``Color(0,0,0)``?

Comment (neikeq) [inline tools/editor/plugins/script_editor_plugin.cpp]: The second parameter is the default value to be used if `text_editor/line_number_color` is not defined in the editor settings: [/tools/editor/editor_settings.cpp#L151-L158](https://github.com/godotengine/godot/blob/57cddf3811b3abfe557aac4ebcf55394b51ee596/tools/editor/editor_settings.cpp#L151-L158)

Comment (bojidar-bg) [inline tools/editor/plugins/script_editor_plugin.cpp]: @akien-mga `Color(0,0,0)` is the default value, if none is found in the settings, while `Color::html(…)` is saved in the settings.

Comment (akien-mga) [inline tools/editor/plugins/script_editor_plugin.cpp]: Thanks :)

**Pull request merged**